docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -115,6 +115,7 @@ The following modules will be removed in Ansible 2.12. Please update your playbo
 * ``katello`` use <https://github.com/theforeman/foreman-ansible-modules> instead.
 * ``github_hooks`` use :ref:`github_webhook <github_webhook_module>` and :ref:`github_webhook_facts <github_webhook_facts_module>` instead.
 * ``digital_ocean`` use :ref `digital_ocean_droplet <digital_ocean_droplet_module>` instead.
+* ``gce`` use :ref `gce_module <gce_module>` instead.
```

Comment:
This line says "use `gce_module` instead" but in the module it says "use `gcp_compute_instance` instead" - we should be consistent in what we recommend, whether it's one or the other or both.

Reply:
Whoops, typo! Great catch!